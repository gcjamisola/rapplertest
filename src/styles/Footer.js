import styled from 'styled-components';

const Footer = styled.div`
    font-weight: bold;
    padding: 10px;
    text-align: center;
`;

export { Footer };