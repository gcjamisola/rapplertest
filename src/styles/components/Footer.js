import styled from 'styled-components';

const Footer = styled.div`
	background: #232d3b;
	box-shadow: 0px 5px 15px 0px #2470cd inset;
	color: darkgrey;
	display: flex;
	font-weight: bold;
	justify-content: center;
	padding: 10px;
	text-align: center;
`;

export { Footer };
