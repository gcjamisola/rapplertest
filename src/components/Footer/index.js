import React, { Component } from 'react';
import PropTypes from 'prop-types';

import { Footer as FooterSC } from '../../styles';

class Footer extends Component {
    render() {
        return (
            <FooterSC>
                Copyright 2018
            </FooterSC>
        );
    }
}

export default Footer;
