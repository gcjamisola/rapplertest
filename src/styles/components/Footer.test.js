import React from 'react';
import styled from 'styled-components';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import {Footer} from './Footer';

test('it works', () => {
  const tree = renderer.create(<Footer />).toJSON()
  expect(tree).toMatchSnapshot()
})
