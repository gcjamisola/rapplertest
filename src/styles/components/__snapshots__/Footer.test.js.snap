// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  background: #232d3b;
  box-shadow: 0px 5px 15px 0px #2470cd inset;
  color: darkgrey;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-weight: bold;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 10px;
  -webkit-text-align: center;
  text-align: center;
}

<div
  className="c0"
/>
`;
